cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -596,6 +597,17 @@ func TestPolicyUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		"Error on mount apply failing": {
+			initState: "old-data",
+			clearDirs: []string{
+				fmt.Sprintf("run/users/%s/mounts", currentUID),
+			},
+			// This generates an error when the used path already exists as a directory
```

Comment:
+ "instead of a file" or something like that in the description

Reply:
Hum, I see +1 from you here, but that wasn’t changed?

Reply:
Should be fixed now